docs/user-guide/ze-cics.md

```diff
@@ -65,8 +67,24 @@ The Zowe Explorer pane will show tree views for **Data Sets**, **Unit System Ser
 If you already have a Zowe CICS CLI profile, the CICS tree will load the default profile on startup.  
 
 If you don't have an existing Zowe CICS CLI profile, follow these steps to create one:
+#### Using Zowe Team Configuration
 
 1. Select the **+** button in the CICS tree.
+
+2. Select the **Create New CICS profile** option to open your config file.
+
+3. Edit the config file to add a CICS profile.
```

Comment:
It might be implicit, but i have been told that the user needs to save the file before the refresh.  Suggest adding a step.

Save the file.